dev-libs/libconfig/libconfig-1.5.recipe

```diff
@@ -0,0 +1,75 @@
+SUMMARY="A simple C/C++ configuration file library"
+DESCRIPTION="Libconfig is a simple library for processing structured \
+configuration files. Libconfig is very compact, a fraction of the size \
+of the expat XML parser library. This makes it well-suited for \
+memory constrained systems like handheld devices."
+HOMEPAGE="http://www.hyperrealm.com/libconfig/"
+COPYRIGHT="2005-2015 Mark A Lindner"
+LICENSE="GNU LGPL v3"
+REVISION="1"
+SOURCE_URI="http://www.hyperrealm.com/libconfig/libconfig-$portVersion.tar.gz"
+CHECKSUM_SHA256="e31daa390d8e4461c8830512fe2e13ba1a3d6a02a2305a02429eec61e68703f6"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+	libconfig = $portVersion compat >= 1
+	lib:libconfig = 1.0.0 compat >= 1
+	lib:libconfig++ = 1.0.0 compat >= 1
+	"
+
+REQUIRES="
+	haiku
+	"
+
+PROVIDES_devel="
+	libconfig = $portVersion compat >= 1
+	devel:libconfig = 1.0.0 compat >= 1
+	devel:libconfig++ = 1.0.0 compat >= 1
+	"
+REQUIRES_devel="
+	libconfig == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:automake
+	cmd:bison
+	cmd:flex
+	cmd:gcc
+	cmd:ld
+	cmd:libtoolize
+	cmd:make
+	cmd:pkg_config
```

Comment:
To allow this recipe to be built for a secondary arch, we need to change some lines in BUILD_PREREQUIRES by adding a $secondaryArchSuffix to gcc, ld, libtoolize and pkg_config:
```
	cmd:gcc$secondaryArchSuffix
	cmd:ld$secondaryArchSuffix
	cmd:libtoolize$secondaryArchSuffix
	cmd:make
	cmd:pkg_config$secondaryArchSuffix
```